hazelcast/src/main/java/com/hazelcast/config/MapConfig.java

```diff
@@ -195,15 +195,6 @@ public MapConfigReadOnly getAsReadOnly() {
         return readOnly;
     }
 
-    public boolean isHotRestartEnabled() {
```

Comment:
@gurbuzali: Even if we keep these ones, still EE won't compile. Because I renamed HotRestartConfig to HotRestartPersistenceConfig and HotRestartConfig is now something else.